go/vt/orchestrator/inst/tablet_dao.go

```diff
@@ -80,15 +80,15 @@ func SwitchPrimary(newPrimaryKey, oldPrimaryKey InstanceKey) error {
 		log.Errore(err)
 		return nil
 	}
-	if _, err := ChangeTabletType(oldPrimaryKey, topodatapb.TabletType_REPLICA); err != nil {
+	if _, err := ChangeTabletType(oldPrimaryKey, topodatapb.TabletType_REPLICA, ReplicaSemiSync(newPrimaryKey, oldPrimaryKey)); err != nil {
```

Comment:
The function name `ReplicaSemiSync` is confusing. I'd be more comfortable if it were called `IsReplicaSemiSync`

Reply:
Changed